<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Profil</class>
 <widget class="QWidget" name="Profil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profil</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(31, 40, 51);
</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>480</y>
     <width>611</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">color: rgb(197, 198, 199);
background-color: rgb(69, 162, 158);
font: 75 9pt &quot;Terminal&quot;;</string>
      </property>
      <property name="text">
       <string>study</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(197, 198, 199);
background-color: rgb(69, 162, 158);
font: 75 9pt &quot;Terminal&quot;;</string>
      </property>
      <property name="text">
       <string>test</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="styleSheet">
       <string notr="true">color: rgb(197, 198, 199);
background-color: rgb(69, 162, 158);
font: 75 9pt &quot;Terminal&quot;;</string>
      </property>
      <property name="text">
       <string>statistics</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="styleSheet">
       <string notr="true">color: rgb(197, 198, 199);
background-color: rgb(186, 0, 0);
font: 75 9pt &quot;Terminal&quot;;</string>
      </property>
      <property name="text">
       <string>exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>30</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(197, 198, 199);
background-color: rgb(69, 162, 158);
font: 75 9pt &quot;Terminal&quot;;</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>571</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Trebuchet MS&quot;;
color: rgb(197, 198, 199);
</string>
        </property>
        <property name="text">
         <string>Прогресс</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Trebuchet MS&quot;;
color: rgb(197, 198, 199);
</string>
        </property>
        <property name="text">
         <string>Скорость печати символов</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Trebuchet MS&quot;;
color: rgb(197, 198, 199);
</string>
        </property>
        <property name="text">
         <string>Скорость печати слов</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Trebuchet MS&quot;;
color: rgb(197, 198, 199);
</string>
        </property>
        <property name="text">
         <string>Почта</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Trebuchet MS&quot;;
color: rgb(197, 198, 199);
</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Trebuchet MS&quot;;
color: rgb(197, 198, 199);
</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Trebuchet MS&quot;;
color: rgb(197, 198, 199);
</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Trebuchet MS&quot;;
color: rgb(197, 198, 199);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>260</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(197, 198, 199);
background-color: rgb(69, 162, 158);
font: 75 9pt &quot;Terminal&quot;;</string>
   </property>
   <property name="text">
    <string>save mail</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(69, 162, 158);
font: 75 9pt &quot;Terminal&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>300</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(186, 0, 0);
font: 75 8pt &quot;Trebuchet MS&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
